static/app/views/performance/queues/destinationSummary/messageSpanSamplesPanel.tsx

```diff
@@ -67,6 +72,38 @@ export function MessageSpanSamplesPanel() {
     ? [query.destination, query.transaction].filter(Boolean).join(':')
     : undefined;
 
+  const handleStatusChange = newStatus => {
```

Comment:
While we do allow implicit any in our codebase, I would rather you add a type so that we know what this function expects as an argument.